pkg/constants/common.go

```diff
@@ -73,6 +73,9 @@ const (
 	StatusAvailable = "available"
 	StatusInUse     = "in-use"
 
+	StatusUnmounted = "unmounted"
```

Comment:
aliyun_provider config is needed in global_config.yaml.
aliyun install package is needed in image/